extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json

```diff
@@ -0,0 +1,158 @@
+          }
+        },
+        "xdm:eventMetrics": {
+          "title": "Metrics to be reported",
+          "$ref":
+            "https://ns.adobe.com/experience/campaign/orchestration/reportingeventmetrics",
+          "description":
+            "The metrics for actions performed during this observation."
+        },
+        "https://ns.adobe.com/experience/campaign/orchestration/action": {
+          "title": "Action on step transition",
+          "type": "object",
+          "description":
+            "Action taken during a step transition happening for an user in state machine",
+          "properties": {
+            "xdm:actionID": {
+              "title": "Action identifier",
+              "type": "string",
+              "format": "uri",
+              "description": "Unique identifier denoting the associated action."
+            },
+            "xdm:type": {
+              "title": "Action Type",
+              "type": "string",
+              "description": "The type of action to be performed.",
+              "meta:enum": {
+                "scheduledNotification":
+                  "This action type allows to specify scheduled notifications and wait for the notifications as incoming events for steps",
+                "httpCall":
+                  "This action type is for a HTTP call on an external system",
+                "actionWithPersonalization":
+                  "This action type describes an action with personalization that will be resolved at runtime for each voyager instance",
+                "parameterizedAction":
+                  "This action type describes an action with parameterization",
+                "sendJourneyNotification":
+                  "This action type is to send notification for another journey",
+                "acsWriter":
+                  "The ACS writer action performs REST calls to an Adobe campaign standard instance to write data",
+                "acsMessageCenter":
+                  "The ACS message center action performs REST calls to an Adobe campaign standard instance to send messages with Message Center"
+              }
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/orchestration/externalevent": {
+          "title": "external event received",
+          "type": "object",
+          "description":
+            "External event received, triggering the state transition",
+          "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+          "properties": {
+            "xdm:eventType": {
+              "title": "Type of the event received",
+              "type": "string",
+              "description": "The type for the external event received",
+              "meta:enum": {
+                "http": "The external event recieved on http endpoint"
+              }
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/orchestration/journey": {
```

Comment:
I forgot to mention this before, but for all of Adobe Campaign, you should use `https://ns.adobe.com/experience/campaign/` as the namespace URI, and then have simple property names such as:

`https://ns.adobe.com/experience/campaign/journey`
`https://ns.adobe.com/experience/campaign/journeyVersion`

Reply:
This also applies to the metric URIs.